src/qibolab/platforms/dummy.py

```diff
@@ -60,23 +60,29 @@ def execute_pulse_sequence(self, sequence, nshots=None, relaxation_time=None):
             results[serial] = copy.copy(results[qubit])
         return results
 
-    def set_attenuation(self, qubit, att):  # pragma: no cover
-        pass
+    def set_attenuation(self, qubit, att):
+        """For DummyPlatform this method is empty given that
+        this platform is not connected to instruments"""
```

Comment:
```suggestion
        """Empty since a dummy platform is not connected to any instrument."""
```

Reply:
But then, should you really implement them?

However, not sure how `DummyPlatform` will survive the refactoring, so fine as is. Keep going, we will reconsider later on.

Reply:
I believe that these methods are necessary given that one of the first purposes of having a dummy was to be able to run qibocal code without having to use an actual qpu.